src/Uno.UWP/Generated/3.0.0.0/Windows.ApplicationModel.Activation/ToastNotificationActivatedEventArgs.cs

```diff
@@ -2,12 +2,12 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.ApplicationModel.Activation
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+	#if false || __IOS__ || NET461 || __WASM__ || __MACOS__
```

Comment:
You can mark the class as implemented for all platforms, even if the class is not used for a particular platform. In effect, the implementation of the class will not diverge between platforms.

Reply:
Class for all, and (two) members also?

Reply:
The whole class

Reply:
"whole class" - do you mean line 5, or line 5, 10, and 60?